runtime/compilers/rillv1/parser_test.go

```diff
@@ -94,6 +94,13 @@ measures:
     expression: count(*)
 first_day_of_week: 7
 first_month_of_year: 3
+available_time_ranges:
+  - P2W
+  - range: P4W
+  - range: P2M
+    comparison_ranges:
+      - P1M
+      - P1Y
```

Comment:
Are these comparison ranges or comparison offsets?

Reply:
These are just for testing. Actual values are pending a decision on the format.

Reply:
I meant that if they are offsets, then I think we should make the key `comparison_offsets` instead :) This sounds like you're comparing e.g. 2 months of revenue vs. 1 month or 1 year of revenue – I'm missing context, but I don't think that's what we're doing?